rust/scripts/wasm_all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Is it possible to convert this to Node.js code? So that people working on Windows can run this script with ease

Reply:
Those are pretty basic. I think we can easily convert them once this lands.